File: task-03/Python/2/Sub-Task2.py
def read_and_write_file(input_file, output_file):
    try:
        with open(input_file, 'r') as infile:
            content = infile.read()
        
        with open(output_file, 'w') as outfile:
            outfile.write(content)
        
        print("Content successfully copied from {} to {}.".format(input_file, output_file))
    
    except FileNotFoundError:
        print("One of the files was not found. Please ensure '{}' and '{}' exist.".format(input_file, output_file))
    except IOError as e:
        print("An IOError occurred: {}".format(e))

if __name__ == "__main__":
    read_and_write_file('input.txt', 'output.txt')

File: task-03/Rust/2/Sub-Task2.rs
use std::fs::File;
use std::io::{self, Read, Write};

fn read_and_write_file(input_file: &str, output_file: &str) -> io::Result<()> {
    let mut input = File::open(input_file)?;
    let mut content = String::new();
    input.read_to_string(&mut content)?;
    
    let mut output = File::create(output_file)?;
    output.write_all(content.as_bytes())?;
    
    println!("Content successfully copied from {} to {}.", input_file, output_file);
    Ok(())
}

fn main() -> io::Result<()> {
    read_and_write_file("input.txt", "output.txt")
}
